packages/features/bookings/lib/getBookingFields.ts

```diff
@@ -251,6 +251,7 @@ export const ensureBookingInputsHaveSystemFields = ({
       type: "multiemail",
       editable: "system-but-optional",
       name: "guests",
+      defaultPlaceholder: "Enter Email...",
```

Comment:
We use translation keys here. It has to go first in en/common.json and then key would be used here. Also, **Enter Email...** doesn't look good. How about using "Email Address" which has a translation _email_address_

cc @PeerRich

Reply:
Hi @hariombalhara, I will change the text and send it. Thanks for the update.

Reply:
I have fixed the issue...